src/BuildScriptGenerator.Common/SdkStorageConstants.cs

```diff
@@ -16,7 +16,7 @@ public static class SdkStorageConstants
         public const string DefaultVersionFilePrefix = "defaultVersion";
         public const string DefaultVersionFileType = "txt";
         public const string VersionsToBuildFileName = "versionsToBuild.txt";
-        public const string ContainerMetadataUrlFormat = "{0}/{1}?restype=container&comp=list&include=metadata&marker={2}";
+        public const string ContainerMetadataUrlFormat = "{0}/{1}?restype=container&comp=list&include=metadata&marker={2}&{3}";
```

Comment:
Generally SAS tokens have a `?` at the beginning, so we should maybe check for its existence and remove it if it is there?